tests/upgrades/conftest.py

```diff
@@ -11,7 +11,7 @@
     Second stage: post_upgrade::
 
         The test marked as post upgrade will be run, any saved data in dependent pre_upgrade stage
-        can be restored, if a dependent test fail, the post upgrade test will be skipped
+        can be restored, if a dependent test fail, the post upgrade test will be errored
```

Comment:
failed

Reply:
No, pytest doesn't allow me to fail the test from the fixture.